app/controllers/checkout_controller.rb

```diff
@@ -78,6 +78,18 @@ def confirm_order
 
     return true if redirect_to_payment_gateway
 
+    # Redeem VINE voucher
+    vine_voucher_redeemer = Vine::VoucherRedeemerService.new(order: @order)
+    unless vine_voucher_redeemer.redeem
+      # rubocop:disable Rails/DeprecatedActiveModelErrorsMethods
```

Comment:
It's not actually related to rails, It's because I used a similar API to rails to handle errors in `VoucherRedeemerService` , ie I store the errors in an `errors` hash. Rubocop can't actually tell the difference and think I am trying to access an `ActiveModelErrors` object directly.